Tests/FnGenTest.php

```diff
@@ -303,7 +304,7 @@ public function testFnFalse() {
     public function testFnCount() {
         $fn = FnGen::fnCount();
         $co = new class implements \Countable {
-            public function count()
+            #[ReturnTypeWillChange] public function count()
```

Comment:
Why can't we type-hint this return type with `int`? The number returned below is a hardcoded integer, so adding `: int` should be fine.

Reply:
Swapped to int return type 6113ecf582b203a9911ae2a438ef735c24accd34